src/ansys/units/quantity.py

```diff
@@ -103,6 +105,43 @@ def __init__(
 
         self._unit = units
 
+        for unit in self._chosen_units:
+            if unit.name != units.name and self.dimensions == unit.dimensions:
+                self._value = self.to(unit).value
+                self._unit = unit
+
+    @classmethod
+    def preferred_units(
+        cls, units: list[Union[Unit, str]], remove: bool = False
+    ) -> None:
+        """
+        Add or remove preferred units.
+
+        Quantities are automatically converted to preferred units when the
+        quantity is initialized. Conversion is always carried out if the base
+        units are consistent with the preferred units.
+
+        Each preferred unit must have unique dimensions. To override units with
+        the same dimensions, the original must first be removed.
+
```

Comment:
To solve: https://github.com/ansys/pyansys-units/pull/204#discussion_r1430513442